packages/engine/Source/Scene/VoxelProvider.js

```diff
@@ -214,17 +203,17 @@ Object.defineProperties(VoxelProvider.prototype, {
 });
 
 /**
- * Requests the data for a given tile. The data is a flattened 3D array ordered by X, then Y, then Z.
- * This function should not be called before {@link VoxelProvider#ready} returns true.
- * @function
+ * Requests the data for a given tile.
+ *
+ * @private
  *
  * @param {object} [options] Object with the following properties:
  * @param {number} [options.tileLevel=0] The tile's level.
  * @param {number} [options.tileX=0] The tile's X coordinate.
  * @param {number} [options.tileY=0] The tile's Y coordinate.
  * @param {number} [options.tileZ=0] The tile's Z coordinate.
  * @privateparam {number} [options.keyframe=0] The requested keyframe.
- * @returns {Promise<Array[]>|undefined} A promise to an array of typed arrays containing the requested voxel data or undefined if there was a problem loading the data.
+ * @returns {Promise<VoxelContent>} A promise resolving to a VoxelContent containing the data for the tile.
```

Comment:
Again, I think `undefined` is still a valid return type here if the request needs to retried next frame.

Reply:
OK but we currently aren't retrying voxel tile requests anywhere. This would require changes to VoxelTraversal.